site/app/controllers/admin/LateController.php

```diff
@@ -280,6 +281,29 @@ public function updateExtension() {
         }
     }
 
+    /**
+     * @AccessControl(role="INSTRUCTOR")
+     * @Route("/courses/{_semester}/{_course}/users/view_latedays", methods={"GET"})
+     * @return RedirectResponse|WebResponse
+     **/
+    public function viewStudentLatedays() {
```

Comment:
I wonder if you @williamjallen  would like viewStudentLatedays function to have native PHP type declaration too.

Reply:
certainly!

Reply:
Did make change